### PR TITLE
add shopping cart example using prodo boilerplate

### PR DESCRIPTION
**Pull request by mkamranhamid**

### CODE REVIEW

File: examples/shopping-cart/public/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="shortcut icon" href="/favicon.ico" />
+
+    <meta name="description" content="Web site created used create-prodo-app">
```
Comment (andrejak) [inline examples/shopping-cart/public/index.html]:
```suggestion
    <meta name="description" content="Web site created using create-prodo-app">
```
(I see this is actually a typo in our template, so I'm also planning to fix that there: https://github.com/prodo-dev/prodo-template/pull/15 )

File: examples/shopping-cart/package.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
+{
```
Comment (andrejak) [inline examples/shopping-cart/package.json]: Maybe add an "author" line in this file with your name :)

**Comment by mkamranhamid** [comment]
made the fixes as mentioned by @andrejak which include `index.html` typo and `author` property in package.json

**Comment by mkamranhamid** [comment]
Thanks @andrejak for merging the PR

**Pull request merged**